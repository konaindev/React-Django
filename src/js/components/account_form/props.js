export const props = {
  steps: [
    { name: "Set Password", isComplete: true },
    { name: "Complete Account", isActive: true }
  ],
  title: "Complete your account",
  subtitle:
    "We need just a bit more information about you to complete your account."
};
