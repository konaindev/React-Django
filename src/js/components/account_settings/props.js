const user = {
  account_id: 1,
  account_name: "Remarkably",
  account_settings_url: "/users/account-settings",
  email: "test@psl.com",
  logout_url: "/users/logout/",
  user_id: "usr_jjzpeyfeshzpaha5"
};

const itemsOrder = ["profile", "lock", "email"];

const rules = [
  {
    label: "Be at least 8 characters",
    key: "length"
  },
  {
    label: "Contain alphabetic characters",
    key: "characters"
  },
  {
    label: "Not match personal information",
    key: "personal"
  },
  {
    label: "Not be a commonly used password",
    key: "used"
  }
];

const company_roles = [
  { label: "Owner", value: "owner" },
  { label: "Developer", value: "developer" },
  { label: "Asset Manager", value: "asset_manager" },
  { label: "Property Manager", value: "property_manager" }
];

const office_options = [
  { label: "Global", value: 1 },
  { label: "National", value: 2 },
  { label: "Regional", value: 3 },
  { label: "Other", value: 4 }
];

const office_countries = [
  { label: "United States of America", value: "USA" },
  { label: "United Kingdom", value: "GBR" }
];

const us_state_list = [
  { label: "Washington", value: "Washington" },
  { label: "New York", value: "New York" }
];

const gb_county_list = [
  { label: "London", value: "London" },
  { label: "Aberdeen", value: "Aberdeen" }
];

const profile = {
  avatar_url:
    "https://lh3.googleusercontent.com/-cQLcFi7r2uc/AAAAAAAAAAI/AAAAAAAAAAA/ACHi3rfoCSVbR8qVruV55uAYdSC-znVn2w.CMID/s96-c/photo.jpg",
  first_name: "Phillip",
  last_name: "McPhillipson",
  title: "Founder",
  phone_country_code: "",
  phone: "",
  phone_ext: "",
  company: { label: "Glacier Associates", value: "123456789" },
  company_roles: ["owner", "asset_manager", "property_manager"],
  office_street: "1730 Minor Avenue",
  office_city: "Lansing",
  office_state: { label: "Michigan", value: "Michigan" },
  office_zip: "11111",
  office_country: { label: "United States of America", value: "USA" },
  office_address: "1730 Minor Avenue, Lansing, MI",
  office_name: "Michigan",
  office_type: 3
};

const tabsOrder = ["portfolio", "group", "property"];

const initialTab = "portfolio";

export const props = {
  rules,
  profile,
  user,
  company_roles,
  office_options,
  itemsOrder,
  tabsOrder,
  initialTab,
  office_countries,
  us_state_list,
  gb_county_list,
  dispatch: () => {}
};
