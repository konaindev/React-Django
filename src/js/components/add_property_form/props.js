export const props = {
  post_url: "/projects/new",
  states: [
    { id: "AL", name: "AL" },
    { id: "GA", name: "GA" },
    { id: "NE", name: "NE" },
    { id: "OR", name: "OR" },
    { id: "WY", name: "WY" }
  ],
  packages: [
    { id: "accelerate", name: "Accelerate" },
    { id: "optimize", name: "Optimize" },
    { id: "ground", name: "Ground Up" },
    { id: "other", name: "Not Sure" }
  ]
};
