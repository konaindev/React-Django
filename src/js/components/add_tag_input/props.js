const suggestedTags = [
  { name: "Tag 1", count: 5 },
  { name: "Test Tag 2", count: 3 },
  { name: "Lake Highland Properties 3", count: 2 },
  { name: "New Tag", count: 1 }
];

export default { suggestedTags };
