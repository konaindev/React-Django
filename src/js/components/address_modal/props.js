export default {
  isOpen: true,
  addresses: {
    suggested_address: {
      office_city: "Washington",
      office_state: "DC",
      office_street: "1600 Pennsylvania Ave NW",
      office_zip: "20500",
      office_country: "United States"
    }
  },
  dispatch_type: "FAKE DISPATCH"
};
