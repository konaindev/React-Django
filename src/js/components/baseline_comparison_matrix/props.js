import {
  baselineReport,
  baselineReportWithoutCompetitors,
  baselineReportWithOneCompetitor
} from "../common_report/props";

export const props = { report: baselineReport };
export const no_competitor_props = { report: baselineReportWithoutCompetitors };
export const one_competitor_props = {
  report: baselineReportWithOneCompetitor
};
