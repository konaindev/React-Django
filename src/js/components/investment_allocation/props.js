export const props = {
  acquisition: {
    expenses: {
      reputation_building: "20400.00",
      demand_creation: "48000.00",
      leasing_enablement: "0.00",
      market_intelligence: "0.00"
    },
    total: "68400.00"
  },
  retention: {
    expenses: {
      reputation_building: "0.00",
      demand_creation: "0.00",
      leasing_enablement: "0.00",
      market_intelligence: "0.00"
    },
    total: "0.00"
  }
};

export const propsFill = {
  acquisition: {
    expenses: {
      reputation_building: "20400.00",
      demand_creation: "48000.00",
      leasing_enablement: "0.00",
      market_intelligence: "0.00"
    },
    total: "68400.00"
  },
  retention: {
    expenses: {
      reputation_building: "0.00",
      demand_creation: "0.00",
      leasing_enablement: "6000.00",
      market_intelligence: "0.00"
    },
    total: "6000.00"
  }
};
