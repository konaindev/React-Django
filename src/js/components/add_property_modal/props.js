import { states } from "../add_property_form/states";

export const props = {
  formProps: {
    post_url: null,
    packages: [
      { id: "accelerate", name: "Accelerate" },
      { id: "optimize", name: "Optimize" },
      { id: "ground", name: "Ground Up" },
      { id: "other", name: "Not Sure" }
    ],
    states
  }
};
