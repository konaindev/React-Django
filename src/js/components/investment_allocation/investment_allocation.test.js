import renderer from "react-test-renderer";

import InvestmentAllocation from "./index";
import { props, propsFill } from "./props";

describe("SectionHeader", () => {
  it("renders empty pie", () => {
    const tree = renderer.create(<InvestmentAllocation {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  it("renders 100% pie", () => {
    const tree = renderer
      .create(<InvestmentAllocation {...propsFill} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
