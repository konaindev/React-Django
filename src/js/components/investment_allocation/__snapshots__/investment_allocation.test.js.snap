// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionHeader renders 100% pie 1`] = `
<div
  className="box-row box-row--normal"
>
  <div
    className="child-wrapper"
    style={
      Object {
        "width": "50.00000%",
      }
    }
  >
    <div
      className="investment-allocation"
    >
      <div
        className="report-section"
      >
        <div
          className="section-header"
        >
          <p
            className="section-header__title"
          >
            acquisition INVESTMENT ALLOCATIONS
          </p>
        </div>
        <div
          className="investment-allocation__body"
        >
          <div
            className="investment-allocation__chart"
          >
            <div
              className="VictoryContainer"
              style={
                Object {
                  "height": "100%",
                  "pointerEvents": "none",
                  "position": "relative",
                  "touchAction": "none",
                  "width": "100%",
                }
              }
            >
              <svg
                aria-labelledby="victory-container-3-title victory-container-3-desc"
                height={400}
                role="img"
                style={
                  Object {
                    "height": "100%",
                    "pointerEvents": "all",
                    "width": "100%",
                  }
                }
                viewBox="0 0 400 400"
                width={400}
              >
                <g>
                  <path
                    d="M1.2246467991473532e-14,-200L0,0Z"
                    role="presentation"
                    shapeRendering="auto"
                    style={
                      Object {
                        "fill": undefined,
                        "padding": 10,
                        "stroke": "transparent",
                        "strokeWidth": 1,
                      }
                    }
                    transform="translate(200, 200)"
                  />
                  <path
                    d="M1.2246467991473532e-14,-200L0,0Z"
                    role="presentation"
                    shapeRendering="auto"
                    style={
                      Object {
                        "fill": undefined,
                        "padding": 10,
                        "stroke": "transparent",
                        "strokeWidth": 1,
                      }
                    }
                    transform="translate(200, 200)"
                  />
                  <path
                    d="M1.2246467991473532e-14,-200A200,200,0,1,1,-190.88100037590152,59.70296220033882L0,0Z"
                    role="presentation"
                    shapeRendering="auto"
                    style={
                      Object {
                        "fill": undefined,
                        "padding": 10,
                        "stroke": "transparent",
                        "strokeWidth": 1,
                      }
                    }
                    transform="translate(200, 200)"
                  />
                  <path
                    d="M-190.88100037590152,59.70296220033882A200,200,0,0,1,1.4089627996560445e-13,-200L0,0Z"
                    role="presentation"
                    shapeRendering="auto"
                    style={
                      Object {
                        "fill": undefined,
                        "padding": 10,
                        "stroke": "transparent",
                        "strokeWidth": 1,
                      }
                    }
                    transform="translate(200, 200)"
                  />
                  <text
                    direction="inherit"
                    dx={0}
                    dy={-2.0300000000000002}
                    x={200}
                    y={160}
                  >
                    <tspan
                      dx={0}
                      style={
                        Object {
                          "fill": "#252525",
                          "fontFamily": "'Gill Sans', 'Gill Sans MT', 'Seravek', 'Trebuchet MS', sans-serif",
                          "fontSize": 14,
                          "letterSpacing": "normal",
                          "padding": 20,
                          "stroke": "transparent",
                        }
                      }
                      textAnchor="middle"
                      x={200}
                    >
                      
                    </tspan>
                  </text>
                  <text
                    direction="inherit"
                    dx={0}
                    dy={-2.0300000000000002}
                    x={200}
                    y={160}
                  >
                    <tspan
                      dx={0}
                      style={
                        Object {
                          "fill": "#252525",
                          "fontFamily": "'Gill Sans', 'Gill Sans MT', 'Seravek', 'Trebuchet MS', sans-serif",
                          "fontSize": 14,
                          "letterSpacing": "normal",
                          "padding": 20,
                          "stroke": "transparent",
                        }
                      }
                      textAnchor="middle"
                      x={200}
                    >
                      
                    </tspan>
                  </text>
                  <text
                    direction="inherit"
                    dx={0}
                    dy={4.97}
                    x={232}
                    y={224}
                  >
                    <tspan
                      dx={0}
                      style={
                        Object {
                          "fill": "#252525",
                          "fontFamily": "'Gill Sans', 'Gill Sans MT', 'Seravek', 'Trebuchet MS', sans-serif",
                          "fontSize": 14,
                          "letterSpacing": "normal",
                          "padding": 20,
                          "stroke": "transparent",
                        }
                      }
                      textAnchor="start"
                      x={232}
                    >
                      70%
                    </tspan>
                  </text>
                  <text
                    direction="inherit"
                    dx={0}
                    dy={4.97}
                    x={168}
                    y={176}
                  >
                    <tspan
                      dx={0}
                      style={
                        Object {
                          "fill": "#252525",
                          "fontFamily": "'Gill Sans', 'Gill Sans MT', 'Seravek', 'Trebuchet MS', sans-serif",
                          "fontSize": 14,
                          "letterSpacing": "normal",
                          "padding": 20,
                          "stroke": "transparent",
                        }
                      }
                      textAnchor="end"
                      x={168}
                    >
                      30%
                    </tspan>
                  </text>
                </g>
              </svg>
              <div
                style={
                  Object {
                    "height": "100%",
                    "left": 0,
                    "position": "absolute",
                    "top": 0,
                    "width": "100%",
                    "zIndex": 99,
                  }
                }
              >
                <svg
                  height={400}
                  style={
                    Object {
                      "height": "100%",
                      "overflow": "visible",
                      "width": "100%",
                    }
                  }
                  viewBox="0 0 400 400"
                  width={400}
                />
              </div>
            </div>
          </div>
          <div
            className="investment-allocation__expenses"
          >
            <div
              className="investment-allocation__row"
            >
              <div
                className="investment-allocation__title investment-allocation__title--reputation-building"
              >
                Reputation Building
              </div>
              <div
                className="investment-allocation__value"
              >
                $20k
              </div>
            </div>
            <div
              className="investment-allocation__row"
            >
              <div
                className="investment-allocation__title investment-allocation__title--demand-creation"
              >
                Demand Creation
              </div>
              <div
                className="investment-allocation__value"
              >
                $48k
              </div>
            </div>
            <div
              className="investment-allocation__row investment-allocation__row--empty"
            >
              <div
                className="investment-allocation__title investment-allocation__title--leasing-enablement"
              >
                Leasing Enablement
              </div>
              <div
                className="investment-allocation__value"
              >
                $0
              </div>
            </div>
            <div
              className="investment-allocation__row investment-allocation__row--empty"
            >
              <div
                className="investment-allocation__title investment-allocation__title--market-intelligence"
              >
                Market Intelligence
              </div>
              <div
                className="investment-allocation__value"
              >
                $0
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="child-wrapper"
    style={
      Object {
        "width": "50.00000%",
      }
    }
  >
    <div
      className="investment-allocation"
    >
      <div
        className="report-section"
      >
        <div
          className="section-header"
        >
          <p
            className="section-header__title"
          >
            retention INVESTMENT ALLOCATIONS
          </p>
        </div>
        <div
          className="investment-allocation__body"
        >
          <div
            className="investment-allocation__chart"
          >
            <div
              className="VictoryContainer"
              style={
                Object {
                  "height": "100%",
                  "pointerEvents": "none",
                  "position": "relative",
                  "touchAction": "none",
                  "width": "100%",
                }
              }
            >
              <svg
                aria-labelledby="victory-container-4-title victory-container-4-desc"
                height={400}
                role="img"
                style={
                  Object {
                    "height": "100%",
                    "pointerEvents": "all",
                    "width": "100%",
                  }
                }
                viewBox="0 0 400 400"
                width={400}
              >
                <g>
                  <path
                    d="M1.2246467991473532e-14,-200L0,0Z"
                    role="presentation"
                    shapeRendering="auto"
                    style={
                      Object {
                        "fill": undefined,
                        "padding": 10,
                        "stroke": "transparent",
                        "strokeWidth": 1,
                      }
                    }
                    transform="translate(200, 200)"
                  />
                  <path
                    d="M1.2246467991473532e-14,-200A200,200,0,1,1,-1.2246467991473532e-14,200A200,200,0,1,1,1.2246467991473532e-14,-200Z"
                    role="presentation"
                    shapeRendering="auto"
                    style={
                      Object {
                        "fill": undefined,
                        "padding": 10,
                        "stroke": "transparent",
                        "strokeWidth": 1,
                      }
                    }
                    transform="translate(200, 200)"
                  />
                  <path
                    d="M-2.1437508791444564e-13,-200L0,0Z"
                    role="presentation"
                    shapeRendering="auto"
                    style={
                      Object {
                        "fill": undefined,
                        "padding": 10,
                        "stroke": "transparent",
                        "strokeWidth": 1,
                      }
                    }
                    transform="translate(200, 200)"
                  />
                  <path
                    d="M-2.1437508791444564e-13,-200L0,0Z"
                    role="presentation"
                    shapeRendering="auto"
                    style={
                      Object {
                        "fill": undefined,
                        "padding": 10,
                        "stroke": "transparent",
                        "strokeWidth": 1,
                      }
                    }
                    transform="translate(200, 200)"
                  />
                  <text
                    direction="inherit"
                    dx={0}
                    dy={-2.0300000000000002}
                    x={200}
                    y={205}
                  >
                    <tspan
                      dx={0}
                      style={
                        Object {
                          "fill": "#252525",
                          "fontFamily": "'Gill Sans', 'Gill Sans MT', 'Seravek', 'Trebuchet MS', sans-serif",
                          "fontSize": 14,
                          "letterSpacing": "normal",
                          "padding": 20,
                          "stroke": "transparent",
                        }
                      }
                      textAnchor="middle"
                      x={200}
                    >
                      
                    </tspan>
                  </text>
                  <text
                    direction="inherit"
                    dx={0}
                    dy={11.969999999999999}
                    x={200}
                    y={195}
                  >
                    <tspan
                      dx={0}
                      style={
                        Object {
                          "fill": "#252525",
                          "fontFamily": "'Gill Sans', 'Gill Sans MT', 'Seravek', 'Trebuchet MS', sans-serif",
                          "fontSize": 14,
                          "letterSpacing": "normal",
                          "padding": 20,
                          "stroke": "transparent",
                        }
                      }
                      textAnchor="middle"
                      x={200}
                    >
                      100%
                    </tspan>
                  </text>
                  <text
                    direction="inherit"
                    dx={0}
                    dy={-2.0300000000000002}
                    x={200}
                    y={205}
                  >
                    <tspan
                      dx={0}
                      style={
                        Object {
                          "fill": "#252525",
                          "fontFamily": "'Gill Sans', 'Gill Sans MT', 'Seravek', 'Trebuchet MS', sans-serif",
                          "fontSize": 14,
                          "letterSpacing": "normal",
                          "padding": 20,
                          "stroke": "transparent",
                        }
                      }
                      textAnchor="middle"
                      x={200}
                    >
                      
                    </tspan>
                  </text>
                  <text
                    direction="inherit"
                    dx={0}
                    dy={-2.0300000000000002}
                    x={200}
                    y={205}
                  >
                    <tspan
                      dx={0}
                      style={
                        Object {
                          "fill": "#252525",
                          "fontFamily": "'Gill Sans', 'Gill Sans MT', 'Seravek', 'Trebuchet MS', sans-serif",
                          "fontSize": 14,
                          "letterSpacing": "normal",
                          "padding": 20,
                          "stroke": "transparent",
                        }
                      }
                      textAnchor="middle"
                      x={200}
                    >
                      
                    </tspan>
                  </text>
                </g>
              </svg>
              <div
                style={
                  Object {
                    "height": "100%",
                    "left": 0,
                    "position": "absolute",
                    "top": 0,
                    "width": "100%",
                    "zIndex": 99,
                  }
                }
              >
                <svg
                  height={400}
                  style={
                    Object {
                      "height": "100%",
                      "overflow": "visible",
                      "width": "100%",
                    }
                  }
                  viewBox="0 0 400 400"
                  width={400}
                />
              </div>
            </div>
          </div>
          <div
            className="investment-allocation__expenses"
          >
            <div
              className="investment-allocation__row investment-allocation__row--empty"
            >
              <div
                className="investment-allocation__title investment-allocation__title--reputation-building"
              >
                Reputation Building
              </div>
              <div
                className="investment-allocation__value"
              >
                $0
              </div>
            </div>
            <div
              className="investment-allocation__row investment-allocation__row--empty"
            >
              <div
                className="investment-allocation__title investment-allocation__title--demand-creation"
              >
                Demand Creation
              </div>
              <div
                className="investment-allocation__value"
              >
                $0
              </div>
            </div>
            <div
              className="investment-allocation__row"
            >
              <div
                className="investment-allocation__title investment-allocation__title--leasing-enablement"
              >
                Leasing Enablement
              </div>
              <div
                className="investment-allocation__value"
              >
                $6k
              </div>
            </div>
            <div
              className="investment-allocation__row investment-allocation__row--empty"
            >
              <div
                className="investment-allocation__title investment-allocation__title--market-intelligence"
              >
                Market Intelligence
              </div>
              <div
                className="investment-allocation__value"
              >
                $0
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`SectionHeader renders empty pie 1`] = `
<div
  className="box-row box-row--normal"
>
  <div
    className="child-wrapper"
    style={
      Object {
        "width": "50.00000%",
      }
    }
  >
    <div
      className="investment-allocation"
    >
      <div
        className="report-section"
      >
        <div
          className="section-header"
        >
          <p
            className="section-header__title"
          >
            acquisition INVESTMENT ALLOCATIONS
          </p>
        </div>
        <div
          className="investment-allocation__body"
        >
          <div
            className="investment-allocation__chart"
          >
            <div
              className="VictoryContainer"
              style={
                Object {
                  "height": "100%",
                  "pointerEvents": "none",
                  "position": "relative",
                  "touchAction": "none",
                  "width": "100%",
                }
              }
            >
              <svg
                aria-labelledby="victory-container-1-title victory-container-1-desc"
                height={400}
                role="img"
                style={
                  Object {
                    "height": "100%",
                    "pointerEvents": "all",
                    "width": "100%",
                  }
                }
                viewBox="0 0 400 400"
                width={400}
              >
                <g>
                  <path
                    d="M1.2246467991473532e-14,-200L0,0Z"
                    role="presentation"
                    shapeRendering="auto"
                    style={
                      Object {
                        "fill": undefined,
                        "padding": 10,
                        "stroke": "transparent",
                        "strokeWidth": 1,
                      }
                    }
                    transform="translate(200, 200)"
                  />
                  <path
                    d="M1.2246467991473532e-14,-200L0,0Z"
                    role="presentation"
                    shapeRendering="auto"
                    style={
                      Object {
                        "fill": undefined,
                        "padding": 10,
                        "stroke": "transparent",
                        "strokeWidth": 1,
                      }
                    }
                    transform="translate(200, 200)"
                  />
                  <path
                    d="M1.2246467991473532e-14,-200A200,200,0,1,1,-190.88100037590152,59.70296220033882L0,0Z"
                    role="presentation"
                    shapeRendering="auto"
                    style={
                      Object {
                        "fill": undefined,
                        "padding": 10,
                        "stroke": "transparent",
                        "strokeWidth": 1,
                      }
                    }
                    transform="translate(200, 200)"
                  />
                  <path
                    d="M-190.88100037590152,59.70296220033882A200,200,0,0,1,1.4089627996560445e-13,-200L0,0Z"
                    role="presentation"
                    shapeRendering="auto"
                    style={
                      Object {
                        "fill": undefined,
                        "padding": 10,
                        "stroke": "transparent",
                        "strokeWidth": 1,
                      }
                    }
                    transform="translate(200, 200)"
                  />
                  <text
                    direction="inherit"
                    dx={0}
                    dy={-2.0300000000000002}
                    x={200}
                    y={160}
                  >
                    <tspan
                      dx={0}
                      style={
                        Object {
                          "fill": "#252525",
                          "fontFamily": "'Gill Sans', 'Gill Sans MT', 'Seravek', 'Trebuchet MS', sans-serif",
                          "fontSize": 14,
                          "letterSpacing": "normal",
                          "padding": 20,
                          "stroke": "transparent",
                        }
                      }
                      textAnchor="middle"
                      x={200}
                    >
                      
                    </tspan>
                  </text>
                  <text
                    direction="inherit"
                    dx={0}
                    dy={-2.0300000000000002}
                    x={200}
                    y={160}
                  >
                    <tspan
                      dx={0}
                      style={
                        Object {
                          "fill": "#252525",
                          "fontFamily": "'Gill Sans', 'Gill Sans MT', 'Seravek', 'Trebuchet MS', sans-serif",
                          "fontSize": 14,
                          "letterSpacing": "normal",
                          "padding": 20,
                          "stroke": "transparent",
                        }
                      }
                      textAnchor="middle"
                      x={200}
                    >
                      
                    </tspan>
                  </text>
                  <text
                    direction="inherit"
                    dx={0}
                    dy={4.97}
                    x={232}
                    y={224}
                  >
                    <tspan
                      dx={0}
                      style={
                        Object {
                          "fill": "#252525",
                          "fontFamily": "'Gill Sans', 'Gill Sans MT', 'Seravek', 'Trebuchet MS', sans-serif",
                          "fontSize": 14,
                          "letterSpacing": "normal",
                          "padding": 20,
                          "stroke": "transparent",
                        }
                      }
                      textAnchor="start"
                      x={232}
                    >
                      70%
                    </tspan>
                  </text>
                  <text
                    direction="inherit"
                    dx={0}
                    dy={4.97}
                    x={168}
                    y={176}
                  >
                    <tspan
                      dx={0}
                      style={
                        Object {
                          "fill": "#252525",
                          "fontFamily": "'Gill Sans', 'Gill Sans MT', 'Seravek', 'Trebuchet MS', sans-serif",
                          "fontSize": 14,
                          "letterSpacing": "normal",
                          "padding": 20,
                          "stroke": "transparent",
                        }
                      }
                      textAnchor="end"
                      x={168}
                    >
                      30%
                    </tspan>
                  </text>
                </g>
              </svg>
              <div
                style={
                  Object {
                    "height": "100%",
                    "left": 0,
                    "position": "absolute",
                    "top": 0,
                    "width": "100%",
                    "zIndex": 99,
                  }
                }
              >
                <svg
                  height={400}
                  style={
                    Object {
                      "height": "100%",
                      "overflow": "visible",
                      "width": "100%",
                    }
                  }
                  viewBox="0 0 400 400"
                  width={400}
                />
              </div>
            </div>
          </div>
          <div
            className="investment-allocation__expenses"
          >
            <div
              className="investment-allocation__row"
            >
              <div
                className="investment-allocation__title investment-allocation__title--reputation-building"
              >
                Reputation Building
              </div>
              <div
                className="investment-allocation__value"
              >
                $20k
              </div>
            </div>
            <div
              className="investment-allocation__row"
            >
              <div
                className="investment-allocation__title investment-allocation__title--demand-creation"
              >
                Demand Creation
              </div>
              <div
                className="investment-allocation__value"
              >
                $48k
              </div>
            </div>
            <div
              className="investment-allocation__row investment-allocation__row--empty"
            >
              <div
                className="investment-allocation__title investment-allocation__title--leasing-enablement"
              >
                Leasing Enablement
              </div>
              <div
                className="investment-allocation__value"
              >
                $0
              </div>
            </div>
            <div
              className="investment-allocation__row investment-allocation__row--empty"
            >
              <div
                className="investment-allocation__title investment-allocation__title--market-intelligence"
              >
                Market Intelligence
              </div>
              <div
                className="investment-allocation__value"
              >
                $0
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="child-wrapper"
    style={
      Object {
        "width": "50.00000%",
      }
    }
  >
    <div
      className="investment-allocation"
    >
      <div
        className="report-section"
      >
        <div
          className="section-header"
        >
          <p
            className="section-header__title"
          >
            retention INVESTMENT ALLOCATIONS
          </p>
        </div>
        <div
          className="investment-allocation__body"
        >
          <div
            className="investment-allocation__chart"
          >
            <div
              className="VictoryContainer"
              style={
                Object {
                  "border": "1px dashed",
                  "borderColor": undefined,
                  "borderRadius": "50%",
                  "height": "100%",
                  "pointerEvents": "none",
                  "position": "relative",
                  "touchAction": "none",
                  "width": "100%",
                }
              }
            >
              <svg
                aria-labelledby="victory-container-2-title victory-container-2-desc"
                height={400}
                role="img"
                style={
                  Object {
                    "height": "100%",
                    "pointerEvents": "all",
                    "width": "100%",
                  }
                }
                viewBox="0 0 400 400"
                width={400}
              >
                <g>
                  <path
                    d="M1.2246467991473532e-14,-200L0,0Z"
                    role="presentation"
                    shapeRendering="auto"
                    style={
                      Object {
                        "fill": undefined,
                        "padding": 10,
                        "stroke": "transparent",
                        "strokeWidth": 1,
                      }
                    }
                    transform="translate(200, 200)"
                  />
                  <path
                    d="M1.2246467991473532e-14,-200L0,0Z"
                    role="presentation"
                    shapeRendering="auto"
                    style={
                      Object {
                        "fill": undefined,
                        "padding": 10,
                        "stroke": "transparent",
                        "strokeWidth": 1,
                      }
                    }
                    transform="translate(200, 200)"
                  />
                  <path
                    d="M1.2246467991473532e-14,-200L0,0Z"
                    role="presentation"
                    shapeRendering="auto"
                    style={
                      Object {
                        "fill": undefined,
                        "padding": 10,
                        "stroke": "transparent",
                        "strokeWidth": 1,
                      }
                    }
                    transform="translate(200, 200)"
                  />
                  <path
                    d="M1.2246467991473532e-14,-200L0,0Z"
                    role="presentation"
                    shapeRendering="auto"
                    style={
                      Object {
                        "fill": undefined,
                        "padding": 10,
                        "stroke": "transparent",
                        "strokeWidth": 1,
                      }
                    }
                    transform="translate(200, 200)"
                  />
                  <text
                    direction="inherit"
                    dx={0}
                    dy={-2.0300000000000002}
                    x={200}
                    y={160}
                  >
                    <tspan
                      dx={0}
                      style={
                        Object {
                          "fill": "#252525",
                          "fontFamily": "'Gill Sans', 'Gill Sans MT', 'Seravek', 'Trebuchet MS', sans-serif",
                          "fontSize": 14,
                          "letterSpacing": "normal",
                          "padding": 20,
                          "stroke": "transparent",
                        }
                      }
                      textAnchor="middle"
                      x={200}
                    >
                      
                    </tspan>
                  </text>
                  <text
                    direction="inherit"
                    dx={0}
                    dy={-2.0300000000000002}
                    x={200}
                    y={160}
                  >
                    <tspan
                      dx={0}
                      style={
                        Object {
                          "fill": "#252525",
                          "fontFamily": "'Gill Sans', 'Gill Sans MT', 'Seravek', 'Trebuchet MS', sans-serif",
                          "fontSize": 14,
                          "letterSpacing": "normal",
                          "padding": 20,
                          "stroke": "transparent",
                        }
                      }
                      textAnchor="middle"
                      x={200}
                    >
                      
                    </tspan>
                  </text>
                  <text
                    direction="inherit"
                    dx={0}
                    dy={-2.0300000000000002}
                    x={200}
                    y={160}
                  >
                    <tspan
                      dx={0}
                      style={
                        Object {
                          "fill": "#252525",
                          "fontFamily": "'Gill Sans', 'Gill Sans MT', 'Seravek', 'Trebuchet MS', sans-serif",
                          "fontSize": 14,
                          "letterSpacing": "normal",
                          "padding": 20,
                          "stroke": "transparent",
                        }
                      }
                      textAnchor="middle"
                      x={200}
                    >
                      
                    </tspan>
                  </text>
                  <text
                    direction="inherit"
                    dx={0}
                    dy={-2.0300000000000002}
                    x={200}
                    y={160}
                  >
                    <tspan
                      dx={0}
                      style={
                        Object {
                          "fill": "#252525",
                          "fontFamily": "'Gill Sans', 'Gill Sans MT', 'Seravek', 'Trebuchet MS', sans-serif",
                          "fontSize": 14,
                          "letterSpacing": "normal",
                          "padding": 20,
                          "stroke": "transparent",
                        }
                      }
                      textAnchor="middle"
                      x={200}
                    >
                      
                    </tspan>
                  </text>
                </g>
              </svg>
              <div
                style={
                  Object {
                    "height": "100%",
                    "left": 0,
                    "position": "absolute",
                    "top": 0,
                    "width": "100%",
                    "zIndex": 99,
                  }
                }
              >
                <svg
                  height={400}
                  style={
                    Object {
                      "height": "100%",
                      "overflow": "visible",
                      "width": "100%",
                    }
                  }
                  viewBox="0 0 400 400"
                  width={400}
                />
              </div>
            </div>
          </div>
          <div
            className="investment-allocation__expenses"
          >
            <div
              className="investment-allocation__row investment-allocation__row--empty"
            >
              <div
                className="investment-allocation__title investment-allocation__title--reputation-building"
              >
                Reputation Building
              </div>
              <div
                className="investment-allocation__value"
              >
                $0
              </div>
            </div>
            <div
              className="investment-allocation__row investment-allocation__row--empty"
            >
              <div
                className="investment-allocation__title investment-allocation__title--demand-creation"
              >
                Demand Creation
              </div>
              <div
                className="investment-allocation__value"
              >
                $0
              </div>
            </div>
            <div
              className="investment-allocation__row investment-allocation__row--empty"
            >
              <div
                className="investment-allocation__title investment-allocation__title--leasing-enablement"
              >
                Leasing Enablement
              </div>
              <div
                className="investment-allocation__value"
              >
                $0
              </div>
            </div>
            <div
              className="investment-allocation__row investment-allocation__row--empty"
            >
              <div
                className="investment-allocation__title investment-allocation__title--market-intelligence"
              >
                Market Intelligence
              </div>
              <div
                className="investment-allocation__value"
              >
                $0
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
